Packs/Vectra_AI/Integrations/VectraDetect/test_data/single_user.json

```diff
@@ -0,0 +1,9 @@
+{
+    "account_type": "local",
+    "authentication_profile": null,
+    "email": "dummy-dev@fake.com",
+    "id": 123,
```

Comment:
```suggestion
    "id": "123",
```

Reply:
To fix https://app.circleci.com/pipelines/github/demisto/content/197794/workflows/df2c79ec-2b7d-490c-81af-bf5e2fcf6383/jobs/581561?invite=true#step-105-7